### ISSUE TITLE
How to use detect.py

### ISSUE THREAD

**Issue by lili2012** [microsoft/YOLaT-VectorGraphicsRecognition#10]
I use 
`CUDA_VISIBLE_DEVICES=0 python -u cad_recognition/train.py --batch_size 4 --data_dir data/FloorPlansGraph5_iter --phase train --lr 2.5e-4 --lr_adjust_freq 9999999999999999999999999999999999999 --in_channels 5 --n_blocks 2 --n_blocks_out 2 --arch centernet3cc_rpn_gp_iter2  --graph bezier_cc_bb_iter --data_aug true  --weight_decay 1e-5 --postname run182_2 --dropout 0.0 --do_mixup 0 --bbox_sampling_step 10`

and get a training model run182_2_best.pth change train.py to detect.py and append --pretrained_model trainedModel/run182_2_best.pth get below command:

`CUDA_VISIBLE_DEVICES=0 python -u cad_recognition/detect.py --batch_size 4 --data_dir data/FloorPlansGraph5_iter  --phase train --lr 2.5e-4 --lr_adjust_freq 9999999999999999999999999999999999999 --in_channels 5 --n_blocks 2 --n_blocks_out 2 --arch centernet3cc_rpn_gp_iter2  --graph bezier_cc_bb_iter --data_aug true  --weight_decay 1e-5 --postname run182_2 --dropout 0.0 --do_mixup 0 --bbox_sampling_step 10  --pretrained_model trainedModel/run182_2_best.pth`

but it shows below errors:

<img width="1002" height="310" alt="Image" src="https://github.com/user-attachments/assets/d1e9360b-f716-4be1-82e3-243d405a2596" />
